packages/appkit/src/client.ts

```diff
@@ -198,6 +201,9 @@ export class AppKit {
       }
     })
     PublicStateController.set({ initialized: true })
+    setTimeout(() => {
+      this.checkExistingConnection()
+    }, 0)
```

Comment:
shouldn't you check this when initializing the auth provider? 
`syncAuthProvider`